rust_src/src/lisp.rs

```diff
@@ -88,17 +87,17 @@ impl LispObject {
 impl LispObject {
     pub fn get_type(self) -> Lisp_Type {
         let raw = self.to_raw() as EmacsUint;
-        let res = (if USE_LSB_TAG {
-                       raw & (!VALMASK as EmacsUint)
-                   } else {
-                       raw >> VALBITS
-                   }) as u8;
-        unsafe { mem::transmute(res) }
+        let res = if USE_LSB_TAG {
+            raw & (!unsafe { VALMASK } as EmacsUint)
```

Comment:
It's a static global, so there's no way for the compiler to verify that it's been initialized.